nucypher/cli/characters/ursula.py

```diff
@@ -186,6 +157,7 @@ def ursula(click_config,
         click_config.emitter(message="Generated keyring {}".format(ursula_config.keyring_dir), color='green')
 
         click_config.emitter(message="Saved configuration file {}".format(ursula_config.config_file_location), color='green')
+        return
```

Comment:
Is this a mistake...? I can't tell.

Reply:
Nope - this terminated CLI execution preventing other command from being run.